docs/GettingStarted.md

```diff
@@ -1,209 +1,368 @@
-    mamba create -n ros_env python=3.11
-    mamba activate ros_env
-
-    # this adds the conda-forge channel to the new created environment configuration 
-    conda config --env --add channels conda-forge
-    # and the robostack-staging channel
-    conda config --env --add channels robostack-staging
-    # if you want to use Jazzy, also add its own channel:
-    conda config --env --add channels robostack-jazzy
-    # remove the defaults channel just in case, this might return an error if it is not in the list which is ok
-    conda config --env --remove channels defaults
-    ```
+    !!! warning "PowerShell is not supported"
 
+        On Windows, Powershell is not supported, only the Command Prompt terminal is supported.
+    
+    ## Installing ros
+    === "Mamba"
+
+        ```bash title="Prepare an environment to use the correct channels"
+        mamba create -n ros_env python=3.11
+        mamba activate ros_env
+
+        # this adds the conda-forge channel to the new created environment configuration 
+        conda config --env --add channels conda-forge
+        # remove the defaults channel just in case, this might return an error if it is not in the list which is ok
+        conda config --env --remove channels defaults
+        ```
+        !!! note
+            There are different channels depending on the version of ROS that you want to install, you must only install ONE version of ROS per environment:
+        === "ROS1 Noetic"
+            ```
+            conda config --env --add channels robostack-noetic
+            mamba install ros-noetic-desktop
+            ```
+        === "ROS2 Humble"
+            ```
+            conda config --env --add channels robostack-humble
+            mamba install ros-humble-desktop
+            ```
+        === "ROS2 Jazzy"
+            ```
+            conda config --env --add channels robostack-jazzy
+            mamba install ros-jazzy-desktop
+            ```
 
-    ```bash title="Install ROS1 or ROS2"
-    # Install ros-noetic into the environment (ROS1)
-    mamba install ros-noetic-desktop
+    
+        ```bash title="Deactivate and reactivate the environment to initialize the configured ROS environment"
+        mamba deactivate
+        mamba activate ros_env
+        ```
 
-    # Install ros-humble into the environment (ROS2 Humble)
-    mamba install ros-humble-desktop
+    === "Micromamba"
 
-    # Install ros-jazzy into the environment (ROS2 Jazzy)
-    mamba install ros-jazzy-desktop
-    ```
+        When you only have micromamba available use the following commands: 
 
-    ```bash title="Reactivate the environment to initialize the ros env"
-    mamba deactivate
-    mamba activate ros_env
-    ```
+        === "ROS1 Noetic"
+            ```
+            # Create a ros-noetic desktop environment
+            micromamba create -n ros_env -c conda-forge -c robostack-noetic ros-noetic-desktop python=3.11
```

Comment:
```suggestion
            micromamba create -n ros_env -c conda-forge -c robostack-noetic ros-noetic-desktop
```

Perhaps we can remove this for consistency with humble and jazzy? As the env in this case is created in one shot (differently from the mamba instructions), there is no reason to specify before hand the python version, and specifying it explicitly here will create us problem if we ever want to update the Python version.